modules/_node-scripts/esbuild/getImportBridgesPlugin.mjs

```diff
@@ -0,0 +1,79 @@
+import {IMPORT_BRIDGE_FILTER, decodeBridgePath} from './getImportBridgePath.mjs';
+import getExportedSymbols from '../util/getExportedSymbols.mjs';
+import getFlatName from '../util/getFlatName.mjs';
+import getPathPrefix from './getPathPrefix.mjs';
+
+export default function getImportBridgesPlugin(globalImports, globalSymbols) {
```

Comment:
I imagine this plugin being a source of confusion in the future, can we add our reasoning in a comment for why the bridge is needed?

Reply:
I've added a comment to this an other files to make esbuild process clearer.

Reply:
done